common/client-libs/gateway-client/src/client.rs

```diff
@@ -69,20 +71,19 @@ impl GatewayClient {
         mixnet_message_sender: MixnetMessageSender,
         ack_sender: AcknowledgementSender,
         response_timeout_duration: Duration,
+        bandwidth_controller: Option<BandwidthController>,
```

Comment:
We do indeed. This is to handle the wasm client which uses this code too. Because the bandwidth controller is not gated by the `coconut` feature anymore, this `Option` substitutes that.